docs/html/development/getting-started.rst

```diff
@@ -49,18 +49,18 @@ pip's tests are written using the :pypi:`pytest` test framework, :pypi:`mock`
 and :pypi:`pretend`. :pypi:`tox` is used to automate the setup and execution of
 pip's tests.
 
+It is preferable to run the tests in **parallel** for better user experience during development,
+as the test suite is very extensive. WIthout the `-n auto` your tests would be running sequentially,
+causing more time to finish.
```

Comment:
This part reads a bit awkward to me, but I can’t point my finger to why.

Reply:
Should I? drop the `running sequentially` part?

Reply:
I'd suggest changing this to...

> since the tests can take a long time to finish when run sequentially.

Reply:
Acknowledged, changes in new commit to reflect the same